File: examples/ex_axis2.jl
x = linspace(0,5,100)
y = exp(x)

# J2G --------------
#
plot(x,y,'-b','linewidth',1.5)
#
set(gca,'yscale','log')
set(gca,'xtick',[0,2.5,5])
set(gca,'ylim',[1 150])

File: examples/octavematlab/ex_axis2.m
% OK APR 9
x = linspace(1,10,100);
y = exp(x)./(x.^3.*log(x+0.5));

plot(x,y,'-','color','olive')
ylim([0 11])
xlim([0 11])
set(gca,'xtick',[2 5 10])
set(gca,'xticklabel',['$\sin(\pi x_0)$','$\mathcal L$','$\alpha^2_1$'])